src/Paramore.Brighter.MessagingGateway.RMQ/RmqMessageGateway.cs

```diff
@@ -103,19 +103,21 @@ public virtual void Dispose()
         /// <param name="queueName">Name of the queue. For producer use default of "Producer Channel". Passed to Polly for debugging</param>
         /// <param name="makeExchange">Do we create the exchange if it does not exist</param>
         /// <returns><c>true</c> if XXXX, <c>false</c> otherwise.</returns>
-        protected void EnsureBroker(string queueName = "Producer Channel", OnMissingChannel makeExchange = OnMissingChannel.Create)
+        protected void EnsureBroker(ChannelName queueName = null, OnMissingChannel makeExchange = OnMissingChannel.Create)
```

Comment:
Could be worth making that nullable

Reply:
I tried. I would need to switch the whole repo to nullable. Worth doing, but I think as we are in "fix the failing tests" it is a separate PR.

Reply:
sounds good to me